airbyte-commons-protocol/src/test/java/io/airbyte/commons/protocol/serde/AirbyteMessageV1SerDeTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.serde;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.AirbyteMessage.Type;
+import io.airbyte.protocol.models.ConnectorSpecification;
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.junit.jupiter.api.Test;
+
+class AirbyteMessageV1SerDeTest {
+
+  @Test
+  void v0SerDeRoundTripTest() throws URISyntaxException {
```

Comment:
why is this called `v0****Test` while the test verifies that V1 Serializer is consistent with V1 Deserializer?

Reply:
because I copy-pasted this 😅 will fix later (probably once I'm off-call)

Reply:
It seems to be a still valid command

Reply:
fixed, thanks